### DISCUSSION TITLE
From 8050 Problem to a New Path for NEET Individuals [Jingyi Zhao]

### DISCUSSION THREAD

**Discussion by SAdeephell0** [realitydeslab/playshop-summer-2024#21] [Proposal]
**Chinese Title**: 从8050问题到NEET群体的新出路
**Research Questions:**
How can transforming NEET individuals help solve the 8050 problem and mitigate the trend of societal aging?

**Keywords:**
NEET, 8050 problem, aging, social participation, mental health

**Artefact/Media:**
- Images

**Abstract:**
The rise of automation and the digital economy has exacerbated the isolation and marginalization of NEET (Not in Education, Employment, or Training) individuals, while the growing NEET population has intensified the societal challenge known as the 8050 problem (80-year-old parents having to care for their 50-year-old unemployed children). To mitigate the trend of societal aging, "Tangible Empathy" aims to construct a simulated community that trains NEET individuals in life skills and work essentials, reintegrating them into society. By leveraging inclusive interactive devices, this project creates a community environment that promotes social inclusion, enhances self-efficacy, and provides psychological support.

The simulated community provides targeted vocational training for NEET individuals, including technical skills, craft skills, and service industry skills, to enhance their employability. By collaborating with local businesses and government agencies, the project offers internships and job placement opportunities, ensuring that NEET individuals can quickly find suitable employment post-training, thus alleviating the economic burden on elderly caregivers.

Using tangible user interfaces and embedded systems, the "Tangible Empathy" simulated community project designs training interactive devices to help NEET individuals engage in meaningful activities, develop new skills, and build social connections. The simulated community will also provide real-time feedback and personalized support, encouraging users to participate in community activities and educational opportunities, thereby reducing the caregiving burden on the elderly. These methods aim to address the 8050 problem, promote the reintegration of NEET individuals into society, enhance overall well-being, infuse new vitality into society, and foster sustainable socio-economic development.

Simulated Community Project Content:

1. Inclusive Interactive Devices:
   - Community Center Interactive Devices: These devices help NEET individuals learn basic life skills, such as financial management, time planning, and household management, through gamification. The devices provide immediate guidance and correction via tactile feedback and visual cues.

2. Personalized Support:
   - Mental Health Support: Online and offline psychological counseling helps NEET individuals overcome learned helplessness and social withdrawal, enhancing their sense of self-efficacy. Various community activities are organized to encourage NEET individuals to actively participate, build new social relationships, and improve their sense of belonging to society.

The simulated community offers targeted vocational training for NEET individuals, focusing on technical skills, craft skills, and service industry skills to enhance their employability. By collaborating with local businesses and government agencies, the project provides internships and job placement opportunities, ensuring that NEET individuals can quickly secure suitable employment post-training. This approach not only alleviates the economic burden on elderly caregivers but also contributes to a more dynamic and inclusive workforce.

1.1 Vocational Training Center

The Vocational Training Center is one of the core areas of the simulated community, offering courses in technical training, craft skills, and service industry skills. By collaborating with local businesses, the center also provides internships and job placement services to help NEET individuals find suitable employment upon completing their training.

 1.2 Mental Health Support Center

The Mental Health Support Center offers personalized psychological counseling for NEET individuals, helping them overcome mental barriers such as learned helplessness and social withdrawal. The center also organizes mental health lectures and support groups, providing emotional support and social connections.

1.3 Community Interaction Zone

The Community Interaction Zone is a place where NEET individuals can participate in social activities and build relationships. Through various interactive devices and gamified activities, the zone helps NEET individuals learn basic life skills, such as financial management and time planning, and provides real-time feedback and guidance.

1.4 Life Skills Training Zone

The Life Skills Training Zone focuses on teaching NEET individuals essential household management skills, such as cooking, cleaning, and shopping. Through hands-on practice and simulated scenarios, the zone helps NEET individuals improve their self-management abilities and independent living skills.

**Visual Reference/Illustrations:**
![IMG_2410](https://github.com/realitydeslab/playshop-summer-2024/assets/173219180/0387fd07-626c-4d2a-a00c-2dcac218a1a4)

How Japan Is Dealing With Hikikomori Problem https://youtu.be/U1TwfeSRDeQ?si=Eeifn43c_udzYuKp

**References:**
1.[ https://doi.org/10.1145/3464385.3464703 ](https://doi.org/10.1145/3464385.3464703)Complementing Studies on Vulnerable Youths with Reddit Data
2.[ https://www.semanticscholar.org/paper/YOUTH-TRANSITIONS%3A-PATTERNS-OF-VULNERABILITY-AND-OF-Furlong-Cartmel/dd45a66a543a000ee0a1eab415ef49f61d26fda2](https://www.semanticscholar.org/paper/YOUTH-TRANSITIONS%3A-PATTERNS-OF-VULNERABILITY-AND-OF-Furlong-Cartmel/dd45a66a543a000ee0a1eab415ef49f61d26fda2) YOUTH TRANSITIONS: PATTERNS OF VULNERABILITY AND PROCESSES OF SOCIAL INCLUSION
3. https://doi.org/10.1111/jpr.12219 Characteristics of Family Interaction of Individuals with Hikikomori (Prolonged Social Withdrawal) From the Viewpoint of Behavior Theory

**Comment by SAdeephell0** [comment]
Tagline: NEETs, stop depending on your parents for support! Take charge of your life, gain new skills, and reintegrate into society for a brighter future.

**Reply by SAdeephell0** [reply]
Experience, learn, and grow. Step out from your secluded home, embrace society, and engage in meaningful interactions. Together, shape your future.

**Comment by SAdeephell0** [comment]
Abstract: "Reshaping the Future: From the '8050' Dilemma to a New Path for NEETs" project employs a simulated community with vocational training centers, mental health support centers, community interaction centers, and life skills training centers to help NEETs reintegrate into society.By offering vocational training, psychological counseling, social activities, and virtual interaction devices, the project aims to improve NEETs' self-efficacy and social belonging. This initiative seeks to alleviate the challenges of an aging society and foster social harmony by enabling NEETs to gain new skills and actively participate in community life.

**Comment by SAdeephell0** [comment]
Uploading Hikikomori reintegrating into society through a simulated community project (vocational training, psychological counseling, virtual interactions).-d544d0.mp4…

**Comment by SAdeephell0** [comment]
keyframe
![IMG_2422](https://github.com/realitydeslab/playshop-summer-2024/assets/173219180/b952e19f-879b-45c1-88ba-1170f6e9ce13)

**Comment by SAdeephell0** [comment]
https://github.com/realitydeslab/playshop-summer-2024/assets/173219180/fec15d72-2fc8-4d08-b362-c33ccad65819

**Comment by glitch-kraft** [comment]
**Title**
From 8050 Problem to a New Path for NEET Individuals

**Tagline**
A social programme designed to overcome hikikomori, the 8050 problem and other challenges of NEET individuals. Through engaging experiences it helps those groups to embrace society through meaningful interactions
